store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/RealStoreBuilder.kt

```diff
@@ -16,30 +16,39 @@ import org.mobilenativefoundation.store.store5.StoreBuilder
 import org.mobilenativefoundation.store.store5.StoreDefaults
 import org.mobilenativefoundation.store.store5.Validator
 
-fun <Key : Any, Input : Any, Output : Any> storeBuilderFromFetcher(
-    fetcher: Fetcher<Key, Input>,
-    sourceOfTruth: SourceOfTruth<Key, *>? = null,
-): StoreBuilder<Key, Output> = RealStoreBuilder(fetcher, sourceOfTruth)
+fun <Key : Any, Network : Any, Output : Any> storeBuilderFromFetcher(
+    fetcher: Fetcher<Key, Network>,
+    sourceOfTruth: SourceOfTruth<Key, Network, Output>? = null,
+): StoreBuilder<Key, Output> =
+    RealStoreBuilder<Key, Network, Output, Network>(fetcher, sourceOfTruth)
 
-fun <Key : Any, Input : Any, Output : Any> storeBuilderFromFetcherAndSourceOfTruth(
-    fetcher: Fetcher<Key, Input>,
-    sourceOfTruth: SourceOfTruth<Key, *>,
-): StoreBuilder<Key, Output> = RealStoreBuilder(fetcher, sourceOfTruth)
+fun <Key : Any, Network : Any, Output : Any> storeBuilderFromFetcherAndSourceOfTruth(
+    fetcher: Fetcher<Key, Network>,
+    sourceOfTruth: SourceOfTruth<Key, Network, Output>,
+): StoreBuilder<Key, Output> =
+    RealStoreBuilder<Key, Network, Output, Network>(fetcher, sourceOfTruth)
 
-fun <Key : Any, Network : Any, Output : Any, Local : Any> storeBuilderFromFetcherSourceOfTruthAndMemoryCache(
+fun <Key : Any, Network : Any, Output : Any> storeBuilderFromFetcherSourceOfTruthAndMemoryCache(
     fetcher: Fetcher<Key, Network>,
-    sourceOfTruth: SourceOfTruth<Key, Local>,
+    sourceOfTruth: SourceOfTruth<Key, Network, Output>,
     memoryCache: Cache<Key, Output>,
-): StoreBuilder<Key, Output> = RealStoreBuilder(fetcher, sourceOfTruth, memoryCache)
+): StoreBuilder<Key, Output> =
+    RealStoreBuilder(fetcher, sourceOfTruth, memoryCache)
 
 internal class RealStoreBuilder<Key : Any, Network : Any, Output : Any, Local : Any>(
     private val fetcher: Fetcher<Key, Network>,
-    private val sourceOfTruth: SourceOfTruth<Key, Local>? = null,
-    private val memoryCache: Cache<Key, Output>? = null
+    private val sourceOfTruth: SourceOfTruth<Key, Local, Output>? = null,
+    private val memoryCache: Cache<Key, Output>? = null,
+    private val converter: Converter<Network, Local, Output> = object :
+        Converter<Network, Local, Output> {
+        override fun fromOutputToLocal(output: Output): Local =
```

Comment:
This `fromOutputToLocal` method is only invoked internally right? Should the error message convey it's not an issue with their usage?